apps/condo/domains/common/components/Comments/Comment.tsx

```diff
@@ -191,7 +196,30 @@ export const Comment: React.FC<ICommentProps> = ({ comment, updateAction, delete
                 </Typography.Text>
             }
             author={comment.user.name}
-            datetime={formatDate(intl, comment.createdAt)}
+            datetime={
+                <div
+                    onMouseOut={() => setDateShowMode('created')}
+                    onMouseOver={() => setDateShowMode('updated')}
+                >
+                    <span
```

Comment:
We have a convention to use `Typography` instead of `span`.

Reply:
@AntonAL Typography has typescript issue: there are no onMouse* methods. Only onClick.

Reply:
You already using `onMouse…` events on `<div>`, just replace `<span>` to `<Typography.Text>`.